src/librustc_span/def_id.rs

```diff
@@ -211,18 +211,20 @@ rustc_data_structures::define_id_collections!(DefIdMap, DefIdSet, DefId);
 /// and a DefId from a different crate would signify a bug somewhere. This
 /// is when LocalDefId comes in handy.
 #[derive(Clone, Copy, PartialEq, Eq, Hash)]
-pub struct LocalDefId(DefIndex);
+pub struct LocalDefId {
+    pub intra_crate_def_index: DefIndex,
```

Comment:
```suggestion
    pub local_def_index: DefIndex,
```
For consistency with other naming.

Reply:
For the record, I first suggested the `intra_crate` prefix in https://github.com/rust-lang/rust/pull/66131#discussion_r345465999, but I guess `local_def_index` is also unique enough to work.

The reason I like "intra" in names is being clearer about what scope something is "local" to, but that's not as important for `DefId`, where "local" already has a clear meaning.

Reply:
`crate_local_def_index`?
I don't know, the current name just doesn't seem to harmonize with the rest of the code to me.